bpfd/src/certs.rs

```diff
@@ -262,6 +262,7 @@ async fn generate_cert(
 
     let subject_alt_name = SubjectAlternativeName::new()
         .dns("localhost,IP:127.0.0.1")
```

Comment:
@Billy99 Does this need to be 
```suggestion
        .dns("IP:127.0.0.1")
```

Now?

Reply:
If we change it, I think it might need to be:
```suggestion
        .dns("localhost,IP:127.0.0.1")
        .dns("localhost")
        .dns("127.0.0.1")
        .dns("::1")
```
I think I saw something similar on StackOverflow, which led me to the original fix, but can't find the link now. But I need someone else to tell me how it should be.

Reply:
I'm alright with the minimal change for now